dev/sg/sg_telemetry.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sourcegraph/sourcegraph/lib/output"
+
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/migration"
+
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/db"
+
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/std"
+	"github.com/urfave/cli/v2"
+)
+
+var telemetryCommand = &cli.Command{
+	Name:     "telemetry",
+	Usage:    "Operations relating to Sourcegraph telemetry",
+	Category: CategoryDev,
+	Subcommands: []*cli.Command{
+		allowlistCommand,
+	},
+}
+
+var allowlistCommand = &cli.Command{
+	Name:  "allowlist",
+	Usage: "Edit the usage data allow list",
+	Flags: []cli.Flag{},
+	Description: `
+Utility that will generate SQL to add and remove events from the usage data allow list.
+https://docs.sourcegraph.com/dev/background-information/data-usage-pipeline#allow-list
+
+Events are keyed by event name and passed in as additional arguments to the add and remove subcommands.
+
+# Generate SQL to add events from the allow list
+sg telemetry allowlist add EVENT_ONE EVENT_TWO
+
+# Generate SQL to remove events from the allow list
+sg telemetry allowlist remove EVENT_ONE EVENT_TWO
+
+# Automatically generate migration files associated with the allow list modification
+sg telemetry allowlist add --migration EVENT_ONE EVENT_TWO
+
+# Provide a specific migration name for the migration files
+sg telemetry allowlist add --migration --name my_migration_name EVENT_ONE EVENT_TWO
+`,
+	Subcommands: []*cli.Command{
+		addAllowlistCommand,
+		removeAllowlistCommand,
+	},
+}
+
+var addAllowlistCommand = &cli.Command{
+	Name:      "add",
+	ArgsUsage: "[event]",
+	Usage:     "Generate the SQL required to add events to the allow list",
+	Description: `
```

Comment:
```suggestion
	UsageText: `
```

this will solve the markdown issue @leonore pointed out, example: context:global repo:^github\.com/sourcegraph/sourcegraph$ file:^dev/sg/sg_ci\.go patternType:standard

Reply:
Thanks @bobheadxi!